### DISCUSSION TITLE
Wat ga je vandaag doen? - 21 juni

### DISCUSSION THREAD

**Discussion by D0nnaz** [Sensinki/Blok-Tech-Team-4A#35] [Wat ga je doen vandaag?]
Gisteren heb ik een beetje aan het MVC model gewerkt, images ook geminified met grunt, heb ik een beginnetje gemaakt voor Rosalie en heb ik aan de WIKI gewerkt. 

Vandaag ga ik denk ik weer aan de WIKI werken, zodat mijn delen af zijn. Verder zag ik nog wat foutjes in de chat, die ga ik fixen en voor de rest Sena helpen met MVC en Sang en Rosalie misschien helpen met liking.

**Comment by dinhs-frontend** [comment]
Rosa en ik gaan kijken of onze feature met elkaar kunnen conecten. En ik ga verder proberen of ik data kan ophalen en in de juiste chat kan zetten.

**Reply by dinhs-frontend** [reply]
Omg het werkt XD

**Reply by D0nnaz** [reply]
![crowd-cheering-crowd](https://github.com/Sensinki/Blok-Tech-Team-4A/assets/92280247/97783592-e097-4236-a043-ec3169f8434e)

**Comment by Sensinki** [comment]
ik was gister bezig met mvc model en vandaag wil ik verder gaan en oplossen
